InterfaceParser/model/function.py

```diff
@@ -0,0 +1,43 @@
+"""Function.
+
+"""
+
+from collections import OrderedDict
+
+from model.interface_item_base import InterfaceItemBase
+
+
+class Function(InterfaceItemBase):
+    """Function.
+
+    :param name: item name
+    :param description: list of string description elements
+    :param design_description: list of string design description elements
+    :param issues: list of issues
+    :param todos: list of string todo elements
+    :param platform: optional platform (string or None)
+    :param scope: optional scope: internal, partner or none (none by defaul, means public)
```

Comment:
Maybe note that the param `scope` only exists in the hmi_api.xml? Also do you know what its purpose is?

Reply:
Notation added. I don't know the purpose cause this model come from legacy sdl_core generator/parser.